forms-flow-ai/forms-flow-ai-ee/forms-flow-bpm/processes/Bundling/subProcess/NOMBundleWorkflow.bpmn

```diff
@@ -151,7 +151,7 @@ task.execution.setVariable('userRole','Site Information Advisor');</camunda:scri
           </camunda:field>
         </camunda:executionListener>
       </bpmn:extensionElements>
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${action == 'Additional Information Requested'}</bpmn:conditionExpression>
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${action == 'Returned'}</bpmn:conditionExpression>
```

Comment:
why is it changed to returned ?

Reply:
But would that display 'Returned' status for the staff users ?

Reply:
no, it will display as Additional Information Requested as we have entry in status table